### PR TITLE
Add init-db for production

### PR DESCRIPTION
**Pull request by alexciornii**

### CODE REVIEW

**Comment by johnsmithm** [comment]
nu ar trebuie si `ENV FLASK_ENV "development"` din dockerfile sa fie in .env?

**Comment by johnsmithm** [comment]
putem crea utilizattorul si in `def create_application():` din fisierul `server.py`, cad ca tot in acest fisier se apeleaza si 
```
if FLASK_ENV == "development":
        seed_cli(app)
```

File: backend/config.py [hunk 1]
```diff
@@ -11,6 +11,6 @@
 SWAGGERUI_BLUEPRINT = get_swaggerui_blueprint(SWAGGER_URL, API_URL, config={"app_name": "Python-Flask-REST"})
 
 DB_NAME = "covid-19"
-DB_HOST = "mongodb://mongodb_container:27017"
+DB_HOST = f"mongodb://{os.environ.get('DB_HOST', 'mongodb_container:27017')}"
```
Comment (johnsmithm) [inline backend/config.py:14]: what about the passwrod and username to access the db? if we use some external db, for production we will not use docker for mongo

Comment (johnsmithm) [reply backend/config.py:14]: for the live the DB_HOST could have the format: mongodb+srv://username:password@clusterid.sadasd.mongodb.net

File: Dockerfile [hunk 2]
```diff
@@ -22,4 +22,4 @@ ADD . /app
 
 EXPOSE 5000
 
-CMD flask run --host=0.0.0.0
+CMD flask run --host=0.0.0.0 && flask init-db
```
Comment (johnsmithm) [inline Dockerfile:25]: this will init-db everytime the container is started?

Comment (alexciornii) [reply Dockerfile:25]: As i remember, yes.

**Comment by johnsmithm** [comment]
changes from this PR was added to the PR 167 and merged

**Pull request closed without merging**